index.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module 'braintree-web-drop-in-react' {
```

Comment:
You don't need to do `declare module` when typing the module itself FYI

Reply:
Thanks, I didn't know that!

Reply:
Can you remove it here?

Reply:
Done.